node-src/tasks/upload.ts

```diff
@@ -170,7 +170,7 @@ export const traceChangedFiles = async (ctx: Context, task: Task) => {
     } else {
       ctx.log.warn(`Could not retrieve dependency changes from lockfiles; checking package.json`);
       const changedPackageFiles = await findChangedPackageFiles(packageManifestChanges);
-      if (changedPackageFiles.length > 0) {
+      if (changedPackageFiles.length > 0 && !ctx.untracedFiles) {
```

Comment:
This doesn't make sense to me. Why do you want to prevent bailing if _any_ file is untraced? We already filter out untraced package.json/lockfiles in `findChangedDependencies`, so that's already respected.